### PR TITLE
feat: list protocols on DefiLlama

### PR DESCRIPTION
**Pull request by Addis-Belete**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/clearpool exports TVL: 
        
 
 
    ethereum-staking          23.99 M
    staking                   23.99 M
    borrowed                  16.16 M
    optimism-borrowed         10.12 M
    ethereum-borrowed         5.55 M
    optimism                  2.40 M
    ethereum                  1.45 M
    mantle-borrowed           316.14 k
    polygon_zkevm-borrowed    174.23 k
    mantle                    93.09 k
    polygon                   41.25 k
    polygon_zkevm             32.35 k
    polygon-borrowed          0
    
    total                    4.02 M

File: projects/clearpool/index.js [hunk 1]
```diff
@@ -1,53 +1,109 @@
...
+  },
+  optimism: {
+    factory: "0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80",
+    fromBlock: 107128813,
+  },
+};
 
-Object.keys(config).forEach(chain => {
-  const { fromBlock, factory } = config[chain]
+const getInfo = async () => {
```
Comment (g1nt0ki) [inline projects/clearpool/index.js:37]: bit confused by this PR, I dont see that function used anywhere. other than that, there is only formatting changes?

Comment (Addis-Belete) [reply projects/clearpool/index.js:37]: @g1nt0ki sry for confusion, we will mark the PR as ready once finished.

**Comment by llamatester** [comment]
Error while running adapter at projects/clearpool: 
        
 
  
    
    
    [object Object]

File: projects/clearpool/index.js [hunk 2]
```diff
@@ -11,44 +11,107 @@ const CPOOL = "0x66761fa41377003622aee3c7675fc7b5c1c2fac5";
...
+      return { pools, tokens };
+    }
+
+    const borrowed = async (api) => {
+      const { pools, tokens } = await _getLogs(api);
+      const bals = await api.multiCall({ abi: borrowFn, calls: pools });
+      api.addTokens(tokens, bals);
+    };
+
+    module.exports[chain] = { tvl, borrowed };
```
Comment (georgeciubotaru) [inline projects/clearpool/index.js:112]:
```suggestion
    module.exports[chain][protocol] = { tvl, borrowed };
```

**Pull request closed without merging**